SA3/qa_web/models.py

```diff
@@ -3,12 +3,52 @@
 
 
 class Post(models.Model):
-    content = models.TextField()
-    owner = models.ForeignKey(User, on_delete=models.SET_NULL, null=True)
+    """
+    Parent meta class that describes a post as a publication on the website with certain content
+    created by a certain owner
+    This will be used for inheritance
+    """
+    content = models.TextField(null=True)
+    owner = models.ForeignKey(User, null= True, on_delete=models.SET_NULL)
     creation_date = models.DateTimeField(auto_now_add=True)
+    upvotes = models.IntegerField(default=0)
+    downvotes = models.IntegerField(default=0)
+    
+    class Meta:
+        abstract = True
 
 class Questions(Post):
-    title = models.CharField(max_length=300)
+    """
+    The question (or thread) initiates the interaction between users' answers,
+    comments, and voting
+    """
+    title = models.CharField(max_length=300, null=True)
```

Comment:
Also in [django document](https://docs.djangoproject.com/en/2.0/ref/models/fields/), it shows us how 'null' works as a field option.  It's suggested that 'Avoid using null on string-based fields such as CharField and TextField. If a string-based field has null=True, that means it has two possible values for “no data”: NULL, and the empty string.'

Reply:
Yes, once again this is true. However, I think that keeping a field to `NULL` (if there was no instantiation of that field) is a good practice. In the case of these string-based field, it might be useful for distinguishing `""` and `NULL` (no data at all).